src/file_resolver.rs

```diff
@@ -72,6 +80,18 @@ impl FileResolver {
         }
     }
 
+    /// Get the import path corresponding to a map
+    pub fn get_import_map(&self, map: &OsString) -> Option<&PathBuf> {
+        match self
+            .import_paths
+            .iter()
+            .find(|(m, _)| m.as_ref() == Some(map))
+        {
+            Some((_, pb)) => Some(pb),
+            None => None,
+        }
```

Comment:
Surely this can be simplified with `.find_map()`

Reply:
ahh didn't know about this, thanks!

Reply:
I made this prettier, didn't use `.find_map()` (wasn't clear how to simplify). Instead I did

```rust
    /// Get the import path corresponding to a map
    pub fn get_import_map(&self, map: &OsString) -> Option<&PathBuf> {
        self.import_paths
            .iter()
            .find(|(m, _)| m.as_ref() == Some(map))
            .map(|(_, pb)| pb)
    }
```

Let me know if you'd like something else